game/main.cpp

```diff
@@ -0,0 +1,33 @@
+// This file is a plaground for testing out the game interface
+
+#include "interface.h"
+
+#include <cstdio>
+#include <cstdint>
+
+V<uint32_t, 0x50C044> debug_mode;
+V<uint32_t, 0xE98EB0> digital_input;
+struct foo {
+  uint32_t x;
+};
+S<foo, 0xF00BAA> s;
+
+C<0, uint32_t(uint32_t, float)> foobar;
+
+void printInput() {
+  printf("input: 0x%08X\n", (uint32_t)digital_input);
+}
+
+#include <thread>
```

Comment:
Anything below this, is experimental and will be kicked out. It's only for testing the interfaces from gdb (`call CreateDebugThread()`)